reference/docs-conceptual/Crescendo/get-started/choose-command-line-tool.md

```diff
@@ -0,0 +1,73 @@
+---
+description: How to select a command-line tool to amplify using Crescendo.
+ms.date: 03/08/2022
+title: Choosing a command-line tool to amplify
+---
+# Choosing the command-line tool for Crescendo
+
+Developing a cmdlet with Crescendo is a rapid, reliable way to _amplify_ a command-line tool to
+produce a cmdlet-like experience. Many times, the tool you need to work with is directly tied to the
+technology you are attempting to automate. This makes the tool choice simple. However, not all tools
+work the same way. To optimize your investment when developing a cmdlet, consider the following
+criteria:
+
+## Command-line tools that make good candidates
+
+- **The original tool is difficult to use.**
+
+  If the tool is simple to use and provides the information you need, there is no need to create a
+  cmdlet. However, this isn't always the case with all command-line tools. Many command-line tools
+  have their own unique syntax, parameters, and output that make it difficult for inexperienced
+  administrators to use the command. Converting the command into a cmdlet provides all the benefits
```

Comment:
My understanding of verbal conversation is that we want to refer to the _thing to be amplified_ as the _tool_ which has one or more _commands_. I'm not sure if the first instance of `command` on this line should be `tool's commands` or `tool`; including options for both ways. I think the distinction I want to make sure we clarify is between `tool`, `command`, `amplified command`, and `cmdlet`.

### Option One

```suggestion
  administrators to use the tool. Converting the tool into a cmdlet provides all the benefits
```

### Option Two

```suggestion
  administrators to use the tool's commands. Converting a tool's command into a cmdlet provides all the benefits
```

Reply:
Some definitions:
- tool - the executable
- command - the use of the tool on the command line (i.e. tool+cmds+params)
- amplified command - a command wrapped in a cmdlet

Reply:
I like option 2 but we don't need to say "tool's commands" in the second sentence. We can just say "command".

Reply:
Agreed - added a third option based on these points.